composer.json

```diff
@@ -15,7 +15,8 @@
         "bringyourownideas/silverstripe-composer-update-checker": "2.1.x-dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
+        "symbiote/silverstripe-queuedjobs": "^4.9"
```

Comment:
Why is this added?

Reply:
Fix this https://github.com/creative-commoners/recipe-reporting-tools/runs/7191481297?check_suite_focus=true#step:10:684
```
 * BringYourOwnIdeas\Maintenance\Model\Package
ERROR [Emergency]: Uncaught Error: Call to undefined method Symbiote\QueuedJobs\Services\QueuedJobService::singleton()
IN GET dev/build
Line 157 in /home/runner/work/recipe-reporting-tools/recipe-reporting-tools/vendor/bringyourownideas/silverstripe-maintenance/src/Model/Package.php
```

Reply:
`QueuedJobService` is `Injectable` as of `4.1.0`.
Please use `^4.1` as the constraint.

Reply:
As [noted](https://github.com/silverstripe/silverstripe-session-manager/pull/112#discussion_r922905438) - phpunit 9 compatibility

Reply:
Didn't notice it's just require-dev, no problem for this one.